src/builder-sdk-config.c

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright © 2018 Codethink Limited
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authors:
+ *       Valentin David <valentin.david@codethink.co.uk>
+ */
+
+#include "builder-sdk-config.h"
+
+#include <json-glib/json-glib.h>
+
+struct BuilderSdkConfig {
+  GObject       parent;
+
+  char         *prefix;
```

Comment:
Is there ever a need to tweak the global prefix? I'm pretty sure that won't even work due to various things being hardcoded. Can we drop this until its neeeded+tested.

Reply:
It felt natural to add it when writing it. But maybe it would not work and I do not mind if I have to remove it.